tests/_fixtures/allureHelper.js

```
@@ -0,0 +1,17 @@
+import { allure as singleton } from 'allure-playwright';
+
+/**
+ * Returns an Allure adapter:
+ *  - testInfo.allure (preferred; satisfies the assignment)
+ *  - otherwise the singleton from allure-playwright
+ *  - otherwise a no-op shim (so tests never crash)
+ */
+export function getAllure(testInfo) {
+  const a = testInfo?.allure ?? singleton;
+  if (a && typeof a.parentSuite === 'function') return a;
+  // no-op shim
+  return new Proxy(
+    {},
+    { get: () => () => {} }
+  );
```

Returning a silent no-op shim (`new Proxy({}, { get: () => () => {} })`) will hide misconfiguration: tests will continue without crashing but metadata calls will be no-ops and Allure reports will miss required attributes. This masks the requirement that "All tests must include Allure hierarchy attributes: parentSuite, suite, and subSuite" and that metadata be attached via the adapter. Instead of a silent shim, either throw an informative error when `testInfo.allure` is missing or at minimum `console.error`/`console.warn` explaining the adapter is not available so you can fix the reporter/fixtures.